slackclient/client.py

```diff
@@ -134,29 +136,6 @@ def rtm_read(self):
         else:
             raise SlackNotConnected
 
-    def rtm_send_message(self, channel, message, thread=None, reply_broadcast=None):
```

Comment:
we need this method back! i'm guessing the implementation just needs to call `self.server.rtm_send_message()` with maybe some ID translation?

Reply:
```                       _ ,___,-'",-=-. 
           __,-- _ _,-'_)_  (""`'-._\ `. 
        _,'  __ |,' ,-' __)  ,-     /. | 
      ,'_,--'   |     -'  _)/         `\ 
    ,','      ,'       ,-'_,`           : 
    ,'     ,-'       ,(,-(              : 
         ,'       ,-' ,    _            ; 
        /        ,-._/`---'            / 
       /        (____)(----. )       ,' 
      /         (      `.__,     /\ /, 
     :           ;-.___         /__\/| 
     |         ,'      `--.      -,\ | 
     :        /            \    .__/ 
      \      (__            \    |_ 
       \       ,`-, *       /   _|,\ 
        \    ,'   `-.     ,'_,-'    \ 
       (_\,-'    ,'\")--,'-'       __\ 
        \       /  // ,'|      ,--'  `-. 
         `-.    `-/ \'  |   _,'         `. 
            `-._ /      `--'/             \ 
               ,'           |              \ 
              /             |               \ 
           ,-'              |               / 
          /                 |             -' 
```